Code/Tools/ProjectManager/Source/ProjectButtonWidget.h

```diff
@@ -140,7 +140,7 @@ namespace O3DE::ProjectManager
         void CopyProject(const ProjectInfo& projectInfo);
         void RemoveProject(const QString& projectName);
         void DeleteProject(const QString& projectName);
-        void BuildProject(const ProjectInfo& projectInfo);
+        void BuildProject(const ProjectInfo& projectInfo, bool skipDialogBox = false);
```

Comment:
This signal was changed but doesn't appear to be used.
```suggestion
        void BuildProject(const ProjectInfo& projectInfo);
```

Reply:
Resolving because it looks like this is connected to `QueueBuildProject` and that is the only use